src/main/java/com/faforever/api/error/ErrorResponse.java

```diff
@@ -1,13 +1,21 @@
 package com.faforever.api.error;
 
+import com.faforever.api.logging.RequestIdFilter;
 import lombok.Data;
+import org.slf4j.MDC;
 
 import java.util.ArrayList;
 
 @Data
 public class ErrorResponse {
+  private final String requestId;
```

Comment:
why dont we leave it as response header?

Reply:
That's probably my personal preference. As part of the payload I have it immediately available e.g. when deserializing the message with a Spring RestTemplate